pxr/usd/usdGeom/schema.usda

```diff
@@ -2353,16 +2382,38 @@ class Camera "Camera" (
         doc = """Frame relative shutter close time, analogous comments from
                  shutter:open apply. A value greater or equal to shutter:open
                  should be authored, otherwise there is no exposure and a
-                 renderer should produce a black image."""
+                 renderer should produce a black image. Used for motion blur."""
     )
     
     # exposure adjustment
     float exposure = 0.0 (
-        doc = """Exposure adjustment, as a log base-2 value.  The default
+        doc = """Exposure compensation, as a log base-2 value.  The default
                  of 0.0 has no effect.  A value of 1.0 will double the
                  image-plane intensities in a rendered image; a value of
                  -1.0 will halve them."""
     )
+
+    # exposure controls
+    float exposure:iso = 100.0 (
+        doc = """The speed rating of the sensor or film when calculating exposure.
+                 Higher numbers give a brighter image, lower numbers darker."""
+    )
+
+    float exposure:time = 1.0 (
+        doc = """Time in seconds that the sensor is exposed to light when calculating exposure.
+                 Longer exposure times create a brighter image, shorter times darker."""
+    )
+
+    float exposure:fNumber = 1.0 (
```

Comment:
f-stop and f-number are synonyms. I used f-number here to distinguish from f-stop, which is used for DOF.

Reply:
t-stop is a measurement used on cinema cameras that is essentially a remapping such that transmission and other geometric effects of the lens are ignored.

Reply:
Since this is namespace anyway, I would consider going with fStop.

I have a larger picture question about what factor in this is taking into account how much light we use because of, e.g., lens coating.

Reply:
> I have a larger picture question about what factor in this is taking into account how much light we use because of, e.g., lens coating.

That's all rolled into "responsivity", which is a catch-all correction factor to model lens transmission loss, ISP highlight protection etc. - basically everything that's too complex to simulate or unknown. For instance, if I shoot a macbeth chart with my DSLR and my 55mm lens, I can compute that responsivity factor as ~0.65 by comparing the pixel values generated by the camera to what a spectrometer gives me.